weblab/entities/manifest.py

```diff
@@ -0,0 +1,82 @@
+import xml.etree.ElementTree as ET
+
+
+MANIFEST_NS = 'http://identifiers.org/combine.specifications/omex-manifest'
+
+
+class ManifestWriter:
+    """
+    Writer for COMBINE archive manifest file
+    """
+    def __init__(self):
+        self._files = []
+
+    def add_file(self, path, fmt, is_master):
+        """
+        Add file to manifest
+
+        :param path: Filename, relative to root of archive
+        :param fmt: File format as string
```

Comment:
We should be using either MIME types or identifiers.org URIs here according to the [OMEX spec](http://co.mbine.org/specifications/omex.version-1.pdf).

I'd be inclined to make the signature `add_file(self, path, *, is_master=False, fmt=None, mimetype=None, combine_format=None)` and treat the last 3 arguments as follows:
* If all `None`, determine a mime type from the file extension (is there a library to do this for us)? and store as `"http://purl.org/NET/mediatypes/" + mime_type`, e.g. `http://purl.org/NET/mediatypes/image/png` for `.png`.
* If `fmt` given, use as-is
* If `combine_format` given, prepend `http://identifiers.org/combine.specifications/`
* If `mimetype` given, prepend `http://purl.org/NET/mediatypes/`
* If more than one given, raise an error!

This is probably overkill for what we need immediately (guessing from file extension would do it I think) but will be useful in the future.